src/pages/Sign/SignIn.js

```diff
@@ -1,9 +1,177 @@
+import axios from 'axios';
 import React from 'react';
 import { withRouter } from 'react-router-dom';
 import styled from 'styled-components';
+import { LOGIN_API } from '../../config';
+
+const kakaoLoginHandler = () => {
+  window.Kakao.Auth.login({
+    success: auth => {
+      console.log(Response);
+      axios(`${LOGIN_API}`, {
+        method: 'GET',
+        headers: {
+          Authorization: auth.access_token,
+        },
+      })
+        .then(result => result.json())
+        .then(result => {
+          console.log(result);
+
+          if (result.TOKEN) {
+            localStorage.setItem('login_kakao_token', result.TOKEN);
+            alert('로그인 되었습니다!');
+          } else {
+            alert('로그인 정보를 다시 확인해주세요');
+          }
+        });
```

Comment:
- axios 사용하실 땐, result.json() 하는 과정을 거치지 않으셔도 됩니다.
```suggestion
      axios(`${LOGIN_API}`, {
        method: 'GET',
        headers: {
          Authorization: auth.access_token,
        },
      })
        .then(result => {
          if (result.TOKEN) {
            localStorage.setItem('login_kakao_token', result.TOKEN);
            alert('로그인 되었습니다!');
          } else {
            alert('로그인 정보를 다시 확인해주세요');
          }
        });
```